### PR TITLE
Get airspeed

### LINKED ISSUE #13: Write and Test Basic Code for Airspeed Data Extraction #5
Given x-plane is running
When the plugin is enabled
Then the plugin outputs airspeed value

Developer Notes:

Write and test a basic code to extract Airspeed from the DataRef in the flight simulator. This will serve as a prototype for extracting data and demonstrate the approach for integrating with the iMotions platform.

Acceptance Criteria:

- [ ] Code that successfully connects to the flight simulator
- [ ]  reads the Airspeed DataRef
- [ ] outputs the Airspeed value 
    - [ ] in X-plane screen
    - [ ] whenever we start the x-plane
    - [ ] when the Airspeed plugin is selected the airspeed value is shown.
Testing Notes:
- [ ] Basic functionality is verified by running the code
- [ ] .xpl file generation
- [ ] checking if the Airspeed data is correctly extracted.

### PR DESCRIPTION
**Pull request by aelnourSLU**
### Fixes #13  

---

### **What was changed?**

- Added True Airspeed indicator to flight data window
---

### **Why was it changed?**

- Completed for continued initialization phase of project

---

### **How was it changed?**

- No major changes, new line added to flight data display window & new dataref object added to collection in file

---

### **Screenshots:**
Highlighted in red are X-Plane's readout (above) and my updated readout (below) showing the flight model's true airspeed, represented in knots.
image

### CODE REVIEW

File: get_altitudes.cpp [hunk 1]
```diff
@@ -79,6 +80,7 @@ PLUGIN_API int XPluginStart(char* outName, char* outSig, char* outDesc) {
     // Obtain datarefs for MSL and AGL elevation, respectively
     elevationMslRef = XPLMFindDataRef("sim/flightmodel/position/elevation");
```
Comment (hrmeiners) [inline get_altitudes.cpp:81]: We need to figure out exactly what DataRefs we need to have in our code. We are confused and going to ask the client about this when we meet this week.

Comment (hrmeiners) [reply get_altitudes.cpp:81]: I would wait to merge this until we receive confirmation about these DataRefs.

**Comment by harshitha-thota** [comment]
Great Work!! This PR adds a True Airspeed indicator to the flight data window, improving user experience during initial project phases. The implementation is clean, and the code is readable, though it needs error handling for cases when the airspeed data reference fails. Overall, it's a solid enhancement that should be ready to merge after addressing these minor suggestions.

File: src/main.cpp
Comment (kate-holdener) [inline src/main.cpp:1]: What's the purpose of this "hello world"?

File: lib/XPLM_64.lib
Comment (kate-holdener) [inline lib/XPLM_64.lib:1]: Was this added by accident?

File: src/main.cpp
Comment (aelnourSLU) [reply src/main.cpp:1]: This was a test file for compilation that I'd forgotten to delete. Thank you, it has been removed.

File: lib/XPLM_64.lib
Comment (aelnourSLU) [reply lib/XPLM_64.lib:1]: This slipped by the gitignore, thank you for catching it. It has been removed.

**Review by harshitha-thota — APPROVED** [review]

**Pull request merged**